sdk/core/core-http/rollup.base.config.js

```diff
@@ -83,7 +83,7 @@ export function browserConfig(test = false, production = false) {
         namedExports: {
           chai: ["assert", "AssertionError", "should", "expect"],
           events: ["EventEmitter"],
-          "@opentelemetry/api": ["CanonicalCode", "SpanKind", "TraceFlags"]
+          "@opentelemetry/api": ["CanonicalCode", "SpanKind", "TraceFlags", "getSpan", "StatusCode"]
```

Comment:
what is `getSpan` used for?

Reply:
I need to go look - the general usage is to get a span when you only have a `Context`. It might have just been me trying to be compatible with some existing code that wanted to pull fields out of a Span. Will be more critical of this in the upcoming commits.